src/askjsx/lib/Let.tsx

```diff
@@ -3,13 +3,18 @@ import * as askjsx from './jsx';
 import { AskCodeOrValue } from '../../askcode';
 askjsx;
 
-export function Set({
-  name,
-  value,
-}: {
+type Props = {
   name: string;
   type?: string;
```

Comment:
What is the "type" here?

Reply:
in `let n: int = 5` it's the value of `int`